draft-ietf-taps-interface.md

```diff
@@ -531,6 +531,12 @@ Transport Properties can have one of a set of data types:
   selection; see {{selection-props}}. When querying, a Preference is of
   type Boolean, with `true` indicating that the Selection Property
   has been applied.
+- Tuple: takes values composed from types defined above.
+  In this this document, it is written as a list in brackets, e.g., ```(Enumeration, Preference)``` 
+  The compsition of types and the order depends on the property and is fix for the property.
+  The actual representation is implementation-dependen.
+- Collection: takes a valiable number of a single of the above types.
```

Comment:
```suggestion
- Collection: A group of one or more value types.
```

Reply:
I like your wording but would prefer to stress that these have to be of the same type.